cns/cniconflist/generator_linux.go

```diff
@@ -32,11 +32,10 @@ func (v *V4OverlayGenerator) Generate() error {
 			cni.NetworkConfig{
 				Type:              overlaycniType,
 				Mode:              cninet.OpModeTransparent,
-				ExecutionMode:     string(util.V4Swift),
```

Comment:
@rbtr Do we want to adjust how v4overlay generator works in this PR? Or leave it as is

Reason I ask, is there's a `testdata/fixtures/azure-linux-swift-overlay.conflist` file already, so I would have to make a new `testdata/fixtures/azure-linux-swift-overlay-consolidated.conflist` if we want a separate PR

Reply:
no, and instead of that move the existing testfile to -v4 and change which files the tests are using

Reply:
I am leaning towards leaving v4overlay generator untouched (adding another file) but would like to know what you think

Reply:
K